### PR TITLE
Requesting a pull to mozilla:v0.5.x from k88hudson:t1593

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/387ce694b03a25d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/387ce694b03a25d/output.txt

Total script time: 1.33 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/cf5a251119238a5/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/cf5a251119238a5/output.txt

Total script time: 1.33 mins

File: templates/supported/shared/plugins/shrink/popcorn.shrink.js [hunk 1]
```diff
@@ -40,10 +40,18 @@
...
       var target,
           text,
-          _popcorn = this;
+          _popcorn = this,
+          type = _popcorn.media.tagName;
 
       options._videoContainer = document.getElementById( options.containerID ) || _popcorn.media;
-      options._videoContainer.classList.add( "shrink" );
+
+      if ( type === "VIDEO" ) {
```
Comment (humphd) [inline templates/supported/shared/plugins/shrink/popcorn.shrink.js:48]: This needs an explanatory comment.  From above it looks like shrink and shrink-alt are the same, so why this dance?

**Comment by jbuck** [comment]
Storycamp is closed

**Pull request closed without merging**